docs/installation/component_overview.md

```diff
@@ -0,0 +1,68 @@
+# Component Overview
+
+A functioning Islandora 8 Stack is made up of dozens of components working in synchronization with each other to store information in your repository, manage that information, and disseminate it intelligently to users. Whether running an installation using the provided Ansible playbook or installing the stack manually, it may be helpful to have a brief overview of all the components we're going to need, in the order we're going to install them, as well as a brief introduction to each component's installation and configuration process.
+
+This list includes four different kinds of components:
+
+- Components which are hard-required (such as Drupal and the Islandora module)
+- Components for which defaults are provided but which can be swapped out (such as the software managing databases, or the repository's storage system)
+- Components that can't easily be swapped out but are not necessarily required (such as using Solr as the site's internal search engine)
+- Components which do not have official alternatives and are not necessarily required, but will likely exist on the vast majority of Islandora 8 installations (such as Alpaca and Crayfish)
+
+## The Webserver Stack - Apache, PHP, and MySQL/PostgreSQL
+
+Combined together, Apache, PHP, and MySQL/PostgreSQL comprise a LAMP or LAPP server used to provide end-user-facing components - namely, the website.
+
+**Apache** is the webserver that will serve up webpages to the public. It will also manage some internal funcionality provided by Crayfish, and will expose Cantaloupe to the public. We’ll be making changes to the VirtualHost entry, enabling some modules, and modifying the ports configuration. The VirtualHost entry will eventually be modified when we need to expose other services like Cantaloupe to the public.
+
+**PHP** is the runtime interpreter for all the code Drupal and Crayfish need to be processed. By default, installing PHP 7.2 will give us a command-line interpreter, as well as an interpreter for Apache. We’re going to install several PHP modules required and/or useful for the components that make use of PHP.
+
+**MySQL** and **PostgreSQL** are database management systems that we will use to store information for many different components like Drupal and Fedora. By default, the Ansible playbook installs MySQL, though this can be switched to PostgreSQL. The manual installation guide recommends and walks through installing and using PostgreSQL.
+
+## The Front-Facing CDM - Composer, Drush, and Drupal
+
+Composer will be used to install both Drupal and Drush simultaneously using the [drupal-project](https://github.com/drupal-composer/drupal-project) repository.
```

Comment:
Any reason why we aren't using the [Islandora Drupal Project repo](https://github.com/Islandora/drupal-project/tree/8.x-1.x)?